less/index.less

```diff
@@ -1,12 +1,258 @@
-// Import your less files here:
+// Variables
+// @desktop: 900px;
+// @tablet: 740px;
+// @mobile: 500px;
 
```

Comment:
I see that you got all of your LESS styles in here great job, but we were looking for the files to imported as it saves you from having to copying the file.